setup.cfg

```diff
@@ -34,4 +34,6 @@ zip_safe = False
 packages = find:
 install_requires =
   numpy
+  pandas<0.25
```

Comment:
Could we add a comment here within setup.cfg so we don't forget it's just for Travis-I was puzzled till I saw the PR notes.
```ini
# FIXME: upper limit on pandas and xarray are just so Travis CI passes python 2.7 tests.
```

Reply:
Good idea.  Adding.